input/pagecontent/reference.md

```diff
@@ -1373,7 +1379,7 @@ To create a mapping, the keywords `Mapping`, `Source`, `Target` and `Id` are use
   Source:   USCorePatient
   Target:   "http://unknown.org/Argonaut-DQ-DSTU2"
   Id:       argonaut-dq-dstu2
```

Comment:
`Description` is also an allowed keyword on mappings now, you will want to add that here.

Reply:
Fixed.